ballerina/client.bal

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+
+# Consists of APIs to integrate with Avro Schema Registry.
+public isolated class Client {
+
+    private handle schemaRegistryClient;
```

Comment:
Why do we need to keep a `handle` to invoke the native functions ? The preferred way is the hide the native objects as native data in the ballerina object itself. 

Refer this [1]

[1] - [https://github.com/ballerina-platform/module-ballerinax-kafka/blob/4a1c27e2acabe3ca6e11ea71e8f15c46804c525c/native/src/main/java/io/ballerina/stdlib/kafka/nativeimpl/consumer/Commit.java#L63](https://github.com/ballerina-platform/module-ballerinax-kafka/blob/4a1c27e2acabe3ca6e11ea71e8f15c46804c525c/native/src/main/java/io/ballerina/stdlib/kafka/nativeimpl/consumer/Commit.java#L63)